deploy/operator-install.yaml

```diff
@@ -15,9 +15,10 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: infinispan-operator
+  creationTimestamp: null
```

Comment:
```suggestion
```

Reply:
Damn that somehow sneaked back in. Fixed.